### DISCUSSION TITLE
Config Files for precompiled bins

### DISCUSSION THREAD

**Discussion by jarboy03** [Jyers/Marlin#1568] [Q&A]
Hi all,

I'm brand new to compiling my own firmware so I apologize if this is an obvious question.  The precompiled bin files are working great on my Ender 3v2 but I've found a few settings that I'd like to tweak (for example, I'd like to make the bed size smaller so I stop knocking my clips off).  I've tried compiling my own firmware and it worked but it seemed like I was missing some of the features that were in the precompiled bin file.  Is there any way to look at the configuration.h and configuration_adv.h files that were used in the precompiled bins?

Thanks!
jarboy03

**Comment by MicroSur** [comment]
https://github.com/Jyers/Marlin/tree/bugfix-JyersUI/Configuration%20Files/E3V2%20Templates

**Reply by jarboy03** [reply]
Thank you!  I was running the UBL BLTouch 15x15 precompiled bin.  Is it safe to assume the UBL BLTouch 10x10 template has all of the same settings and only the number of points was changed?

**Reply by Cal-LF** [reply]
@jarboy03  Yes, you should get those features. You can change the 10x10 or leave it as is.
